pattoo/db/schema/favorite.py

```diff
@@ -45,6 +47,11 @@ class Meta:
         interfaces = (graphene.relay.Node,)
 
 
+class ProtectedFavorite(graphene.Union):
```

Comment:
Add a docstring to explain what you are trying to do here. It will help with maintainability.

Reply:
@Dev-Dominic